README.md

```diff
@@ -12,39 +12,18 @@ You can still find the (archived) previous version of Hyrise on [Github](https:/
 
 ## Citation
 
-When referencing this version of Hyrise, please use the following bibtex entry:
-<details><summary>(click to expand)</summary>
-  
+When referencing this version of Hyrise, please use the following bibtex entry: 
 ```bibtex
-@inproceedings{DBLP:conf/edbt/DreselerK0KUP19,
-  author    = {Markus Dreseler and
-               Jan Kossmann and
-               Martin Boissier and
-               Stefan Klauck and
-               Matthias Uflacker and
-               Hasso Plattner},
-  editor    = {Melanie Herschel and
-               Helena Galhardas and
-               Berthold Reinwald and
-               Irini Fundulaki and
-               Carsten Binnig and
-               Zoi Kaoudi},
-  title     = {Hyrise Re-engineered: An Extensible Database System for Research in
-               Relational In-Memory Data Management},
-  booktitle = {Advances in Database Technology - 22nd International Conference on
-               Extending Database Technology, {EDBT} 2019, Lisbon, Portugal, March
-               26-29, 2019},
+@inproceedings{DBLPlike:conf/edbt/DreselerK0KUP19,
```

Comment:
Is there a need to alter the DBLP key?

Reply:
Not really, I just thought the original one is too cluttered and I did not want to keep the key when it’s no longer “the original”.

Maybe key `hyrise_reengineered`?

Reply:
> We could as well just remove the DBLP: prefix

I like that.

Zündung?

Reply:
sure